sql/mysqld.cc

```diff
@@ -8835,6 +8836,16 @@ struct my_option my_long_options[] = {
      "to replicate-do-db.",
      nullptr, nullptr, nullptr, GET_STR, REQUIRED_ARG, 0, 0, 0, nullptr, 0,
      nullptr},
+    {"replica-enable-event", OPT_REPLICA_ENABLE_EVENT,
+     "Tells the replication applier thread to enable the events that match "
+     "the specified wildcard pattern without setting it as SLAVESIDE_DISABLED. "
+     "To specify more than one event, use the directive multiple times, once "
+     "for each event. This will work for cross-database events. Example: "
+     "replica-enable-event=foo%.bar% will enable the events in all "
+     "databases on replica server that start with 'foo' and whose event names "
+     "start with 'bar'.",
```

Comment:
Sorry, one last minute doubt. Should we mention that, this option should be used on events that are read-only? :thinking:

Reply:
I think that is implied that using this feature with non-read-only events will cause inconsistency. But however, let us do it during the documentation phase.

Reply:
Nothing in JIRA, github description, commit message tells about using this option for READ-ONLY events ;)

I feel, user has access to this help text immediately without opening docs etc. Lets extend the last line. Anyway, it didn't reach 80 chars :).  "Should be used for read-only events"

Reply:
@venkatesh-prasad-v can you please reply to the above comment?

Reply:
I'm sorry. I missed a ping for your last comment.

Sure. I will add a line in the end that it is recommended to use this feature only for read-only events to avoid data inconsistencies.

Reply:
Thanks @venkatesh-prasad-v :+1: